recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ outputs:
   - name: unidist-all
     requirements:
       host:
-        - python >=3.7.1,<3.12
+        - python >=3.7.1
```

Comment:
@YarShev  Looks like you need to remove any pins for python in `host` and `run` section.

Looks pretty much the same: https://github.com/bioconda/bioconda-recipes/pull/30938

Reply:
Let me try.